docs/CONTRIBUTING.md

```diff
@@ -134,18 +134,10 @@ let expected = {% if test.expected is object -%}
 {%- endif %};
 ```
 
-If every test case needs to do some crunching of the inputs,
-you can add utils functions at the top of the tera template.
-See [`word-count`'s template][word-count-tmpl] for an example.
```

Comment:
This is a piggy-back, I removed this recommendation because of reasons explained in #1824